ruby_on_rails/vcr.md

```diff
@@ -1,8 +1,56 @@
 # VCR
 
-VCR is a gem for recording HTTP requests to external services and replaying them. A VCR setup should be very specifically tied to
-the tests which need VCR because it's quite expensive.
+VCR is a gem for recording HTTP requests to external services and replaying them.
+A VCR setup should be very specifically tied to the tests which need VCR because it's quite expensive.
 
-Here's an article on about some tweaks which can be made to a VCR setup: https://blog.arkency.com/3-tips-to-tune-your-vcr-in-tests/
+Here's an example configuration we use at Renuo:
+
+```rb
+# spec/rails_helper.rb
+
+RSpec.configure do |config|
+  # …
+
+  # Disable VCR completely for tests that are not tagged with :vcr
+  config.around do |example|
+    if example.metadata[:vcr]
+      example.run
+    else
+      VCR.turned_off { example.run }
+    end
+  end
+end
+
+# WebMock catches everything per default, we allow localhost for Capybara/Selenium
```

Comment:
and maybe also to not mock away internal requests?

Reply:
That's a good point. The current configuration is maybe not precise enough. We would allow HTTP calls to ourselves.
Be aware though that for example request specs are not actual HTTP requests.